src/frontends/ir/src/utils.cpp

```diff
@@ -98,4 +98,18 @@ bool get_dimension_from_attribute(const pugi::xml_node& node, const std::string&
     return true;
 }
 
+void str_to_set_of_strings(const std::string& value, std::set<std::string>& res) {
+    std::stringstream ss(value);
+    std::string field;
+    while (getline(ss, field, ',')) {
+        // trim leading and trailing whitespaces
+        auto strBegin = field.find_first_not_of(" ");
+        if (strBegin == std::string::npos)
+            IE_THROW() << "Cannot get vector of parameters! \"" << value << "\" is incorrect";
+        auto strRange = field.find_last_not_of(" ") - strBegin + 1;
```

Comment:
For case like that:
```cpp
        set<std::string> res;
        str_to_set_of_strings(" ", res);
        assert(res == std::set<std::string>({}));
```  
it will throw an exception.

Reply:
Hi I suppose " " is a incorrect case

Reply:
We need to track this behavior, and throw an exception with human-readable message or handle this case

Reply:
@ilyachur, fixed